File: Solutions/20249/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int main() 
{
	int n, m, a, sum=0, zarfiat;
	
	cin >> n >> m ;
	
	zarfiat = n*m;
	
	for ( int i = 0 ; i < n ; ++i )
	{
	    cin >> a;
	    
	    sum += a;
	}
	
	cout << ( zarfiat - sum )/m;
	return 0;
}

File: Solutions/20249/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner (System.in);
        int n = scan.nextInt() , k = scan.nextInt();
        int i , capacity = 0,tmp;
        for(i = 0 ; i < n ; i++) {
            tmp = scan.nextInt();
            capacity+= tmp;
        }
        int res = capacity/k;
        if(capacity%k != 0)
            res++;
        System.out.print(n - res);
    }
}